src/features/txnDelegation/SignHotspot.tsx

```diff
@@ -166,13 +166,30 @@ const SignHotspot = () => {
   }, [solana.transferData])
 
   const title = useMemo(() => {
-    if (
-      solana.transactions.onboardIotHotspotV0?.gatewayAddress ||
-      solana.transactions.onboardMobileHotspotV0?.gatewayAddress
-    ) {
-      return t('signHotspot.title')
+    const hasIotOnboard =
+      !!solana.transactions.onboardIotHotspotV0?.gatewayAddress
```

Comment:
Is it possible for transactions to be null or undefined?

Reply:
There will always at least been an empty object there.